data-processing/entities/citations/types.py

```diff
@@ -19,6 +19,9 @@ class Bibitem(SerializableEntity):
     text: str
     " Plaintext extracted for bibitem. "
 
+    bibitem_code: str
+    ' Bibitem code'
```

Comment:
Tiny thing: is there a meaning behind the description of the field above being enclosed in double quotes, and this description being enclosed in single quotes? If not, can we have them be consistent with each other (otherwise I think it'll make people think they're different for a reason).

Reply:
(TBH  both of these actually confuse me - I would have expected a comment, but I trust that you and the writer of the other one know what you're doing... Either way though, I think consistency between the two would be great unless there's a reason they're different.)